app/models/manageiq/providers/ansible_tower/automation_manager.rb

```diff
@@ -27,6 +27,8 @@ class ManageIQ::Providers::AnsibleTower::AutomationManager < ManageIQ::Providers
   require_nested :Refresher
   require_nested :RefreshWorker
 
+  has_many :configuration_workflows, :dependent => :destroy, :foreign_key => "manager_id", :inverse_of => :manager
```

Comment:
@Ladas see this as well
This is due to the removal of the relationship from parent class https://github.com/ManageIQ/manageiq/pull/17720/files